bundler/spec/dependabot/bundler/update_checker/conflicting_dependency_resolver_spec.rb

```diff
@@ -44,10 +42,8 @@
   let(:lockfile) do
     Dependabot::DependencyFile.new(content: lockfile_body, name: "Gemfile.lock")
   end
-  let(:gemfile_body) { fixture("ruby", "gemfiles", "subdep_blocked_by_subdep") }
-  let(:lockfile_body) do
-    fixture("ruby", "lockfiles", "subdep_blocked_by_subdep.lock")
-  end
+  let(:gemfile_body) { fixture("projects", "bundler1", "subdep_blocked_by_subdep", "Gemfile") }
+  let(:lockfile_body) { fixture("projects", "bundler1", "subdep_blocked_by_subdep", "Gemfile.lock") }
```

Comment:
I like this as a way to move a little bit towards projects, but without having to cut over the entire spec. Nice one 👍